libcontainer/cgroups/fs/memory.go

```diff
@@ -34,9 +32,15 @@ func (s *MemoryGroup) Apply(d *cgroupData) (err error) {
 				return err
 			}
 		}
-		// We have to set kernel memory here, as we can't change it once
-		// processes have been attached to the cgroup.
-		if err := s.SetKernelMemory(path, d.config); err != nil {
+		// We have to limit the kernel memory here as it won't be accounted at all
```

Comment:
We need to have a check here for determining if kmem is enabled on the system.
@hqhq @vishh Suggestions on how to check for the same?